src/calendar/header/style.ts

```diff
@@ -1,9 +1,10 @@
 import {StyleSheet, Platform} from 'react-native';
+// @ts-expect-error
 import * as defaultStyle from '../../style';
 
 const STYLESHEET_ID = 'stylesheet.calendar.header';
 
-export default function (theme = {}) {
+export default function (theme: any = {}) {
```

Comment:
here too. Why not object type?

Reply:
Changing it to object causes a ts error since it's trying to get `theme[STYLESHEET_ID]` but it doesn't recognize it from the general object type. 
I can change it to `_.get(theme, STYLESHEET_ID)` but I'm not sure it's better.. wdyt?

Reply:
Since this will repeat itself all over the library I suggest we'll define a `theme` type.